README.md

```diff
@@ -2,12 +2,22 @@
 
 This project is the combination of all demos hosted on https://privacybydesign.foundation/demo/.
 
+### Preliminaries
+Make sure you have an [IRMA server](https://irma.app/docs/irma-server/) running. This project needs one to handle its IRMA sessions.
+
 ### Install
-* Run `composer install`
-* Run `yarn install`
-* Set IRMA server URL and API token of the IRMA server of your choice in `config.php`.
-* Follow the instructions in `data`.
```

Comment:
TODO: instructions in `data` are still needed when using JWTs. I should add that.